lib/engine/game/g_1862/map.rb

```diff
@@ -14,120 +14,139 @@ module Map
           {
             'count' => 2,
             'color' => 'green',
-            'code' => 'town=revenue:20;path=a:0,b:_0;path=a:2,b:_0;path=a:1,b:3',
+            'hidden' => 1,
+            'code' => 'town=revenue:20,loc:0;path=a:0,b:_0;path=a:2,b:_0;path=a:1,b:3',
           },
           '16_1b' =>
           {
             'count' => 2,
             'color' => 'green',
-            'code' => 'town=revenue:20;path=a:1,b:_0;path=a:3,b:_0;path=a:0,b:2',
+            'hidden' => 1,
```

Comment:
Why is this tile still visible in the "new tile rendering" screenshot? Is that a screenshot of the `/tiles/1862/all` page, whereas the tile manifest tab in a game would be different?

Reply:
Exactly.

Reply:
I'm not sure it's a bug or a feature.

Reply:
I might consider making a separate section for "hidden" tiles on the `/tiles/*/all` page, or maybe sorting them to the end or something. Not a big deal for now though.